pros/cli/conductor.py

```diff
@@ -159,7 +159,7 @@ def upgrade(ctx: click.Context, project: c.Project, query: c.BaseTemplate, **kwa
     """
     analytics.send("upgrade-project")
     if not query.name:
-        for template in project.templates.keys():
+        for template in tuple(project.templates.keys()):
```

Comment:
Just to be sure, this won't break if they have old conductor files?

Reply:
From my understanding, this function doesn't touch the conductor file (there are no `.save()` calls).